packages/components/List/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import SimpleCard from '../SimpleCard';
+import SearchBar from '../SearchBar';
+
+const List = ({ items, handleSearch, handleEdit, handleDelete }) => {
+
+    const includesText = (textOne, textTwo) => {
+        return textOne.toLowerCase().trim().includes(textTwo.toLowerCase().trim());
+    };
+
+    const [list, handleUpdateList] = useState(items);
```

Comment:
```suggestion
    const [items, setItems] = useState(items);
```

I would keep the naming consistent per React guidelines.
Also change `list` to `items`, if you are using `useState(items)`

Reply:
Cool. This complained because items is already passed in as a const but have done some renaming so that the hook naming is consistent.